src/py42/sdk/queries/fileevents/filters/exposure_filter.py

```diff
@@ -130,3 +139,19 @@ class WindowTitle(FileEventFilterStringField):
     """
 
     _term = u"windowTitle"
+
+
+class RiskIndicatorMimeType(FileEventFilterStringField):
+    """Class that filters events if file type mismatch the file's mime type."""
+
+    _term = u"mimeTypeMismatch"
+
+    FILE_MISMATCH = "true"
+
+
+class RiskIndicatorActiveHours(FileEventFilterStringField):
```

Comment:
you should use `QueryFilterBooleanField` here, see `Shared` for an example. No need to add any constants.

Reply:
Changed.